packages/server/database/rethinkDriver.ts

```diff
@@ -231,7 +231,11 @@ const getRethink = async () => {
   }
   // this is important because pm2 will restart the process & for whatever reason r isn't always healthy
   await r.waitForHealthy()
-  return (r as unknown) as ParabolR
+  return r as unknown as ParabolR
+}
+
+export const closeRethink = async () => {
+  await (await promise)?.drain()
```

Comment:
little nit, but we'll want to drain the whole pool, not just the connection (r.getPoolMaster()?.drain())